app/javascript/components/common/Accordion.tsx

```diff
@@ -45,7 +46,12 @@ const AccordionHeader = ({
       aria-controls={`accordion-panel-${id}`}
       id={`accordion-header-${id}`}
     >
-      {children}
+      {isOpen ? (
+        <GraphicalIcon icon="minus-circle" />
+      ) : (
+        <GraphicalIcon icon="plus-circle" />
+      )}
```

Comment:
Maybe you could add a test for this please?

Reply:
I think this is fine. We already have a test for the correct aria attributes in the JS. We can't also assert on the tests unless we change this to an `<Icon />`and add the correct alt attribute. The alt attribute might be redundant as we already have supplied the correct aria attributes.